src/mixins/Language.js

```diff
@@ -0,0 +1,14 @@
+
+import texts from './../texts.js';
+export default {
+
+    methods: {
+        lang(t) {
+            if (typeof this.texts === 'undefined' || typeof this.texts[t] === 'undefined') {
+                return texts[t];
```

Comment:
What is the return value? What does `texts[t]` mean?

Reply:
it's something i've relocated.
it's for language purposes